### PR TITLE
Theme for spinner, minimap zoom level.

### PR DESCRIPTION
**Pull request by michaelkirk**
## New Day

Screen Shot 2021-02-24 at 7 29 25 PM

## Night

Screen Shot 2021-02-24 at 7 28 41 PM

## Legacy Day

Screen Shot 2021-02-24 at 7 28 14 PM

### CODE REVIEW

File: map_gui/src/tools/minimap.rs [hunk 1]
```diff
@@ -101,38 +101,35 @@ impl<A: AppLike + 'static, T: MinimapControls<A>> Minimap<A, T> {
...
+                    level_btn
+                        .clone()
+                        .image_color(ctx.style().btn_outline.fg_disabled, ControlState::Default)
                 } else {
-                    app.cs().minimap_selected_zoom
+                    level_btn.clone()
                 };
-                let rect = Polygon::rectangle(20.0, 8.0);
-
-                let default_batch = GeomBatch::from(vec![(color, rect.clone())]);
```
Comment (michaelkirk) [inline map_gui/src/tools/minimap.rs:116]: This was a cute trick, but ultimately I thought it looked better to style the "zoom increments" like a normal button, and the easiest way to do that was to make a simple svg rectangle.

File: map_gui/src/tools/minimap.rs [hunk 2]
```diff
@@ -101,38 +101,35 @@ impl<A: AppLike + 'static, T: MinimapControls<A>> Minimap<A, T> {
         let zoom_col = {
             let mut col = vec![ctx
                 .style()
-                .btn_plain_icon("system/assets/speed/speed_up.svg")
+                .btn_plain_icon("system/assets/speed/plus.svg")
```
Comment (michaelkirk) [inline map_gui/src/tools/minimap.rs:104]: just renamed the file - presumably a legacy name, but it's not currently used anywhere else.

File: widgetry/src/event_ctx.rs [hunk 3]
```diff
@@ -168,6 +168,7 @@ impl<'a> EventCtx<'a> {
                 self.style
                     .loading_tips
                     .clone()
+                    .default_fg(Color::WHITE)
```
Comment (michaelkirk) [inline widgetry/src/event_ctx.rs:171]: Unrelated fix: because the loading mostly doesn't leverage the theming stuff, and just has a bunch of hardcoded colors, it also needs to have a hardcoded text color, otherwise it's not legible in the new day mode.

One day we should probably theme the loading screen too.

File: map_gui/src/tools/minimap.rs (hunk 1 above)
Comment (dabreegster) [reply map_gui/src/tools/minimap.rs:116]: The less custom buttons, the better!

**Review by dabreegster — APPROVED** [review]
It's awesome to see the cleanup of all these widgets go so quickly! And a new day is indeed rising...

**Comment by michaelkirk** [comment]
> It's awesome to see the cleanup of all these widgets go so quickly!

Yeah! I think part of it is that we now have pretty good tooling for buttons and 99% of our controls are just thinly disguised buttons.
